packages/front-end/components/Experiment/ChanceToWinColumn.tsx

```diff
@@ -80,7 +80,11 @@ export default function ChanceToWinColumn({
     )} threshold`;
     className = "lost";
   }
-  if (belowMinChange && (chanceToWin > ciUpper || chanceToWin < ciLower)) {
+  if (
+    enoughData &&
```

Comment:
What happens if there's not enough data? Line 124 below suggests we should render <NotEnoughData>, but this will never happen because we've blocked that path with the `enoughData` check here.